Missing file darshan-base-ld-opts.in

Got an error during make.
```
Making all in ld-opts
make[6]: *** No rule to make target 'darshan-base-ld-opts', needed by 'darshan-ld-opts'.  Stop.
```
File 'darshan-runtime/share/ld-opts/darshan-base-ld-opts.in' is missing.

I just tried to reproduce this and hit a problem also, but in my case it was a different problem:

```
make[6]: Entering directory '/home/carns/working/src/darshan/darshan/build-wk/darshan-runtime/lib'
make[6]: *** No rule to make target '../../../darshan-runtime/../modules/autoperf/apxc/darshan-apxc-log-format.h', needed by 'distdir-am'.  Stop.
make[6]: Leaving directory '/home/carns/working/src/darshan/darshan/build-wk/darshan-runtime/lib'
```

--- a/darshan-runtime/share/ld-opts/Makefile.am
+++ b/darshan-runtime/share/ld-opts/Makefile.am
@@ -61,6 +61,7 @@ if BUILD_APMPI_MODULE
 	echo '@$(datadir)/ld-opts/autoperf-apmpi-ld-opts' >> $@
 endif
 
-EXTRA_DIST = darshan-hdf5-ld-opts.in \
+EXTRA_DIST = darshan-base-ld-opts \
+             darshan-hdf5-ld-opts.in \
              darshan-stdio-ld-opts.in
 

